bun.lock

```diff
@@ -48,14 +48,11 @@
         "redaxios": "^0.5.1",
         "semver": "^7.6.3",
         "tempy": "^3.1.0",
-        "tscircuit": "^0.0.574-libonly",
```

Comment:
we want this to be libonly to avoid having a circular dependency with @tscircuit/cli (npm ls @tscircuit/cli should find no copies in this repo)